README.md

```diff
@@ -285,29 +285,28 @@ needs your explicit permission to access your data.
 
 This is an exhaustive description of the reason the script needs each of the permissions:
 
-- **Managing your Google Contacts**  
-  This lets the script access information about your contacts (names, email
-  addresses, birthdays). The script will not modify any of your contacts.
-- **Managing and viewing you calendars**  
-  This lets the script access your birthday and events calendar. The script
-  will get the events from this calendar only and will never modify any event.
-- **Google Plus - Knowing your age, language and email addresses**  
+- **Know your age range and language + View your email addresses**
```

Comment:
You're missing the two trailing spaces here to force a newline.

Reply:
Thanks